lib/workers/repository/init/config.ts

```diff
@@ -140,8 +140,11 @@ export async function mergeRenovateConfig(
   config: RenovateConfig
 ): Promise<RenovateConfig> {
   let returnConfig = { ...config };
-  const repoConfig = await detectRepoFileConfig();
-  checkForRepoConfigError(repoConfig);
+  let repoConfig: RepoFileConfig = {};
+  if (!config.onboarding) {
```

Comment:
doesn't this potentially have other repercussions? I had expected some evaluation added closer to the dashboard logic

Reply:
I think merging onboarding config is the wrong thing, but yes maybe it can break something.